README.md

```diff
@@ -43,20 +43,18 @@ Origin can also be used on its own, in several different configurations:
    shutdown once it takes control.
 
  - The [origin-start example] uses `no_std` and `no_main`, and lets origin
-   start the program using its own program entrypoint. This version must be
-   compiled with an explicit `--target=` flag, even if it's the same as the
-   host, because it uses special RUSTFLAGS, and an explicit `--target` flag
-   prevents those flags from being passed to build scripts. origin handles
-   program and thread startup and shutdown and no part of libc is used.
+   start the program using its own program entrypoint. origin handles program
+   and thread startup and shutdown and no part of libc is used. This is the
+   approach that [origin-studio] uses.
 
  - The [origin-start-no-alloc example] is like origin-start, but disables the
```

Comment:
`origin-start-no-alloc example` doesn't actually link to anything, I'm assuming it was just a little oversight and not intentional?